### ISSUE TITLE
Safari: Missing visual focus and cannot tab on category headers

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#5035]
<img width="1182" alt="Screen Shot 2019-10-16 at 1 40 25 PM" src="https://user-images.githubusercontent.com/45409515/66944104-8bfde900-f01a-11e9-8e6d-c0611050db1b.png">

**Comment by kacperknapik** [comment]
duplicate
